vector/src/main/java/im/vector/app/features/location/UrlMapProvider.kt

```diff
@@ -16,52 +16,22 @@
 
 package im.vector.app.features.location
 
-import im.vector.app.features.raw.wellknown.getElementWellknown
-import org.matrix.android.sdk.api.extensions.tryOrNull
-import org.matrix.android.sdk.api.raw.RawService
-import org.matrix.android.sdk.api.session.Session
+import im.vector.app.features.raw.wellknown.ElementWellKnown
 import javax.inject.Inject
+import javax.inject.Singleton
 
-class UrlMapProvider @Inject constructor(
-        private val session: Session,
-        private val rawService: RawService,
-        locationSharingConfig: LocationSharingConfig,
-) {
-    private val keyParam = "?key=${locationSharingConfig.mapTilerKey}"
+@Singleton
+class UrlMapProvider @Inject constructor() {
 
-    private val fallbackMapUrl = buildString {
-        append(MAP_BASE_URL)
```

Comment:
Maybe we should remove the constants `MAP_BASE_URL` and `STATIC_MAP_BASE_URL` since there are no more used now.